enhancements/update/channel-rename-generally-available.md

```diff
@@ -0,0 +1,354 @@
+
+#### Installer exposure
+
+The installer will need to bump their manually-maintained [default channel][installer-default-channel].
+In the absence of this bump, newly installed clusters in 4.y that do not include a `stable-4.y` channel following the old pattern will have the `RetrievedUpdates=False` and `CannotRetrieveUpdates` alerting discussed in [the version skew section](#version-skew-strategy) until their admin notices, makes their way to the channel documentation, and uses `oc adm upgrade channel fleet-approved-4.16` or similar to select a channel compatible with their current release.
+I don't think the web-console channel changer will support this recovery, because it needs working update service retrieval to populate its drop-down list of available channel choices.
+
+#### Web-console exposure
+
+As described [here](#historical-context), the web-console is mostly out of the business of hard-coding channel information.
+However, they do have [placeholder and help text in the channel model][web-console-channel-modal] that would need updating.
+Both are only presented when the ClusterVersion's `status.desired.channels` is empty, which limits exposure to clusters which have already picked an invalid channel or cleared their channel, or when the update service access is failing.
+In the absence of this bump, they could mislead cluster administrators into setting old-pattern channel names, and the admins would have to follow the link to [channel docs][channel-docs] to discover appropriate values for their 4.y release.
+
+#### CI exposure
+
+In [the release repository][release]:
+
+```console
+release $ git grep -oh 'channel:.*' ci-operator/config | sort | uniq -c
+    128 channel: candidate
+    423 channel: fast
+      1 channel: fast # candidate, fast, stable, eus
+     32 channel: stable
+```
+
+References to `fast` and `stable` for 4.16 and later will need updating, although references to those channels for 4.15 and earlier do not need updating.
+
+Some CI docs will also need updating, including examples like [this][ci-docs-release-channel-example].
+
+Brief searching did not turn up any exposure in [ci-tools][], but it may also have some references that need updating.
+
+#### ART exposure
+
+ART may want to rename their channel-based mirror paths like [`fast-4.14`][mirror-fast-4.14] and [`stable`][mirror-stable].
+Even if they opt to preserve those names, they will probably need to update the tools managing them to track the new channel names.
+
+#### Labs-graph exposure
+
+https://access.redhat.com/labs/ocpupgradegraph/ will need to update its hard-coded list of known channels.
+Alternatively, the set of known channels could [become an update service API][channels-api].
+
+#### Managed ClusterImageSet exposure
+
+The current ClusterImageSet configuration assumes the current channel-naming pattern will continue, and it will need updates to understand the pivot to the new names.
```

Comment:
There's a dependency on OCM/Cluster-Service here. If CS is not ready to accept the new names, changes in CIS will likely get ignored.